css-images-4/Overview.bs

```diff
@@ -1897,15 +1894,90 @@ Color Stop “Fixup”</h4>
 		</pre>
 	</div>
 
-
-
+<!--
+ ██████  ████████ ████████  ████ ████████  ███████  ██████    ███ ███
+██    ██    ██    ██     ██  ██  ██     ██ █       ██    ██  ██     ██
+██          ██    ██     ██  ██  ██     ██ █       ██       ██       ██
+ ██████     ██    ████████   ██  ████████  █████    ██████  ██       ██
+      ██    ██    ██   ██    ██  ██        █             ██ ██       ██
+██    ██    ██    ██    ██   ██  ██        █       ██    ██  ██     ██
+ ██████     ██    ██     ██ ████ ██        ███████  ██████    ███ ███
+-->
 
 1D Image Values: the ''stripes()'' notation {#stripes}
 ======================================================
 
-	Issue: <a href="https://github.com/w3c/csswg-drafts/issues/2532">Per WG resolution</a>,
-	define the ''stripes()'' function
-	which creates a 1D image for use in borders/outlines.
+	While most <<image>> values represent a 2-dimensional image,
+	and <<color>> can be thought of as a "0-dimensional" image
+	(unvarying in either axis),
+	there are some contexts where a 1-dimensional image makes sense,
+	specifying colors along a <dfn export>paint line</dfn>
+	without defining a <em>direction</em> for the line.
+	The <<1d-image>> type represents such images.
+	
+	<pre class=prod>
+		<dfn>&lt;1d-image></dfn> = <<stripes()>>
+		<dfn>stripes()</dfn> = stripes( <<color-stripe>># )
+		<dfn>&lt;color-stripe></dfn> = <<color>> && [ <<length-percentage>> | <<flex>> ]?
+	</pre>
+	
+	The ''stripes()'' function defines a 1d-image
+	as a number of colored stripes with different widths.
+	
+	Each comma-separated entry defines a solid-color stripe,
+	each placed end-to-end on the [=paint line=] in the order given,
+	with the specified <<color>> and thickness.
+	If the thickness is omitted,
+	it defaults to ''1fr''.
+	
+	In each entry, a <<percentage>> is relative to the |painting area|
```

Comment:
I think this is very confusing, the "painting area" is never defined but since it's an area it seems to be 2D. How to resolve a 1D `<length-percentage>` relatively to a 2D area?

Should this refer to the "length of the paint line" instead?

Reply:
https://github.com/w3c/csswg-drafts/issues/8160